docs/cristin.ttl

```diff
@@ -1,57 +1,299 @@
+    rdfs:domain  :Result .
+
+:VARBEID_SPRAK
+    a            rdf:Property ;
+    rdfs:comment """
+    A language that this work has been published in, the respective title, and the respective abstract
+    and an indication whether it is the original language or a translation.
+    """ ;
+    rdfs:range   :VARBEID_SPRAK_Class ;
+    rdfs:domain  :Result .
+
+:sparkkode
+    a            rdf:Property ;
+    rdfs:comment "The language of the title and the abstract included in the VARARBEID_SPRAK entry" ;
+    rdfs:comment "The range could possibly be an xsd:language" ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VARBEID_SPRAK_Class .
+
+:tittletekst
+    a            rdf:Property ;
+    rdfs:comment "The title in the VARARBEID_SPRAK entry. It is supposed to be in the language described by :sprakkode" ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VARBEID_SPRAK_Class .
+
+:sammendragtekst
+    a            rdf:Property ;
+    rdfs:comment "The abstract in the VARARBEID_SPRAK entry. It is supposed to be in the language described by :sprakkode" ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VARBEID_SPRAK_Class .
+
+:status_original
+    a            rdf:Property ;
+    rdfs:comment """
+    Indicator of whether the title in the respective VARBEID_SPRAK is the original title or not.
+    Known possible values are 'J'. """ ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VARBEID_SPRAK_Class .
+
+:VITENSKAPELIGARBEID_LOKAL
+    a            rdf:Property ;
+    rdfs:comment "UNKONWN property" ;
+    rdfs:domain  :Result .
+
+:VARBEID_PERSON
+    a            rdf:Property ;
+    rdfs:comment "A contributor to the respective result entry." ;
+    rdfs:range   :VARBEID_PERSON_Class ;
+    rdfs:domain  :Result .
+
+:personlopenr
+    a            rdf:Property ;
+    rdfs:comment "The Cristin Person identifier of the contributor" ;
+    rdfs:range   xsd:positiveInteger ;
+    rdfs:domain  :VARBEID_PERSON_Class .
+
+:fornavn
+    a            rdf:Property ;
+    rdfs:comment "The first/given name of the contributor of the respective result entry" ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VARBEID_PERSON_Class .
+
+:etternavn
+    a            rdf:Property ;
+    rdfs:comment "The surname/family-name of the contributor of the respective result entry" ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VARBEID_PERSON_Class .
+
+:VARBEID_PERSON_STED
+    a            rdf:Property ;
+    rdfs:comment """
+    Details about the institution which the contributor belongs to and the contributors role to the institution
+    at the time the publication was registered in Cristin.
+    """ ;
+    rdfs:range   :VARBEID_PERSON_STED_Class ;
+    rdfs:domain  :VARBEID_PERSON_Class .
+
+:institusjonsnr
+    a            rdf:Property ;
+    rdfs:comment " The Cristin institution identifier of the respective institution" ;
+    rdfs:range   xsd:positiveInteger ;
+    rdfs:domain  :VARBEID_PERSON_STED_Class .
+
+:avdnr
```

Comment:
While you probably know this, I guess this is avdelingsnummer, ditto underavdelingsnummer in the next property. Or are you using `Unknown property` in a specific way?